### PR TITLE
ahkuma/Staff is redirected to careers page

### PR DESCRIPTION
**Pull request by ahkuma**

### CODE REVIEW

File: src/templates/static/about/index.jsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ const Index = () => (
                             <BoxInner className='border-right-top' image='debt-free' text={it.L('Debt-free')} />
                         </Box>
                         <Box>
-                            <BoxInner className='border-right-top' href='/careers' image='staff' text={it.L('Over [_1] <a href=\'[_2]\'>staff</a> and contractors worldwide', '200', it.url_for('careers'))} />
+                            <BoxInner className='border-right-top' href='/binary-in-numbers' image='staff' text={it.L('Over [_1] <a href=\'[_2]\'>staff</a> and contractors worldwide', '200', it.url_for('binary-in-numbers'))} />
```
Comment (aaron-binary) [inline src/templates/static/about/index.jsx]: Maybe we could use `{it.url_for('binary-in-numbers')}` for the `href`

File: src/javascript/static/pages/static_pages.js [hunk 2]
```diff
@@ -46,4 +46,7 @@ module.exports = {
         onLoad  : () => { MenuSelector.init(['general', 'account-management', 'referral-tools']); },
         onUnload: () => { MenuSelector.clean(); },
     },
+    BinaryInNumbers: {
+        onLoad: () => { Scroll.scrollToHashSection(); },
```
Comment (negarn) [inline src/javascript/static/pages/static_pages.js]: the first time that this page loads, e.g. in a new incognito tab instance, it is not able to scroll to the correct section. perhaps the images load after the scrolling is done and pushes the page down due to that. I suggest adding a small timeout to overcome that. if you have a better solution, that's fine too.

File: src/javascript/app/base/binary_pages.js [hunk 3]
```diff
@@ -145,6 +145,7 @@ const pages_config = {
     'video-facility'         : { module: VideoFacility,              is_authenticated: true, only_real: true },
     'why-us'                 : { module: WhyUs },
     'telegram-bot'           : { module: TelegramBot,                is_authenticated: true },
+    'binary-in-numbers'      : { module: StaticPages.BinaryInNumbers },
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: this should be alphabetical, so on line 118

File: src/javascript/app/base/binary_pages.js [hunk 4]
```diff
@@ -50,7 +50,8 @@ const TNCApproval             = require('../pages/user/tnc_approval');
 const VideoFacility           = require('../pages/user/video_facility');
 
 // ==================== static ====================
-// const Charity            = require('../../static/pages/charity');
+/*
+const Charity            = require('../../static/pages/charity'); */
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: why this change?

File: src/javascript/app/base/binary_pages.js [hunk 5]
```diff
@@ -65,86 +66,89 @@ const WhyUs              = require('../../static/pages/why_us');
 
 /* eslint-disable max-len */
 const pages_config = {
-    account_transfer         : { module: AccountTransfer,            is_authenticated: true, only_real: true, needs_currency: true },
     accounts                 : { module: Accounts,                   is_authenticated: true, needs_currency: true },
     api_tokenws              : { module: APIToken,                   is_authenticated: true },
+    authenticate             : { module: Authenticate,               is_authenticated: true, only_real: true },
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: `u` is after `s`, so this should remain on line 73. we sort alphabetically by keys, and `authenticate` comes after `asset_indexws` but before `authorised_appsws`. the placement of `account_transfer` is wrong, it should be on line 68 as before. since `account_` comes before `accounts`

File: src/javascript/app/base/binary_pages.js [hunk 6]
```diff
@@ -65,86 +66,89 @@ const WhyUs              = require('../../static/pages/why_us');
...
     assessmentws             : { module: FinancialAssessment,        is_authenticated: true, only_real: true },
     asset_indexws            : { module: AssetIndexUI },
-    authenticate             : { module: Authenticate,               is_authenticated: true, only_real: true },
+    account_transfer         : { module: AccountTransfer,            is_authenticated: true, only_real: true, needs_currency: true },
     authorised_appsws        : { module: AuthorisedApps,             is_authenticated: true },
+    cfds                     : { module: GetStarted.CFDs },
+    careers                  : { module: StaticPages.Careers },
+    contact                  : { module: Contact },
+    /*
+    charity                  : { module: Charity }, */
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: again, please leave the comment as a one-liner as before

File: src/javascript/app/base/binary_pages.js [hunk 7]
```diff
@@ -65,86 +66,89 @@ const WhyUs              = require('../../static/pages/why_us');
...
     accounts                 : { module: Accounts,                   is_authenticated: true, needs_currency: true },
     api_tokenws              : { module: APIToken,                   is_authenticated: true },
+    authenticate             : { module: Authenticate,               is_authenticated: true, only_real: true },
     assessmentws             : { module: FinancialAssessment,        is_authenticated: true, only_real: true },
     asset_indexws            : { module: AssetIndexUI },
-    authenticate             : { module: Authenticate,               is_authenticated: true, only_real: true },
+    account_transfer         : { module: AccountTransfer,            is_authenticated: true, only_real: true, needs_currency: true },
     authorised_appsws        : { module: AuthorisedApps,             is_authenticated: true },
+    cfds                     : { module: GetStarted.CFDs },
+    careers                  : { module: StaticPages.Careers },
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: `ca` is beofre `cf`, I'll just move this card back and you can re-check the alphabetical ordering of all these.

File: src/javascript/app/base/binary_pages.js (hunk 4 above)
Comment (ahkuma) [reply src/javascript/app/base/binary_pages.js]: Mainly, because I thought it look prettier because it's aligned other imports

Comment (negarn) [reply src/javascript/app/base/binary_pages.js]: that's fine, better go with consistency and use `//` for single-line comments

Comment (ahkuma) [reply src/javascript/app/base/binary_pages.js]: sure

File: src/javascript/app/base/binary_pages.js [hunk 8]
```diff
@@ -73,38 +73,48 @@ const pages_config = {
     authenticate             : { module: Authenticate,               is_authenticated: true, only_real: true },
     authorised_appsws        : { module: AuthorisedApps,             is_authenticated: true },
     cashier                  : { module: Cashier },
+    careers                  : { module: StaticPages.Careers },
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: `car` is before `cas`

File: src/javascript/app/base/binary_pages.js [hunk 9]
```diff
@@ -73,38 +73,48 @@ const pages_config = {
...
     cashier_passwordws       : { module: CashierPassword,            is_authenticated: true, only_real: true },
-    change_passwordws        : { module: ChangePassword,             is_authenticated: true },
     // charity                  : { module: Charity },
+    cfds                     : { module: GetStarted.CFDs },
+    change_passwordws        : { module: ChangePassword,             is_authenticated: true },
     contact                  : { module: Contact },
+    cyberjaya                : { module: StaticPages.Locations },
+    cryptocurrencies         : { module: GetStarted.Cryptocurrencies },
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: `cr` is definitely before `cy`

File: src/javascript/app/base/binary_pages.js [hunk 10]
```diff
@@ -73,38 +73,48 @@ const pages_config = {
...
     maltainvestws            : { module: FinancialAccOpening,        is_authenticated: true },
     market_timesws           : { module: TradingTimesUI },
+    metals                   : { module: GetStarted.Metals },
     metatrader               : { module: MetaTrader,                 is_authenticated: true, needs_currency: true },
     multi_barriers_trading   : { module: MBTradePage,                needs_currency: true },
-    payment_agent_listws     : { module: PaymentAgentList,           is_authenticated: true },
-    payment_methods          : { module: Cashier.PaymentMethods },
     platforms                : { module: Platforms },
+    payment_methods          : { module: Cashier.PaymentMethods },
+    payment_agent_listws     : { module: PaymentAgentList,           is_authenticated: true },
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: `payment_a` is before `payment_m`

File: src/javascript/app/base/binary_pages.js [hunk 11]
```diff
@@ -73,38 +73,48 @@ const pages_config = {
...
     professional             : { module: professionalClient,         is_authenticated: true, only_real: true },
+    profit_tablews           : { module: ProfitTable,                is_authenticated: true, needs_currency: true },
     realws                   : { module: RealAccOpening,             is_authenticated: true },
     redirect                 : { module: Redirect },
     regulation               : { module: Regulation },
     reset_passwordws         : { module: ResetPassword,              not_authenticated: true },
     securityws               : { module: Settings,                   is_authenticated: true },
-    self_exclusionws         : { module: SelfExclusion,              is_authenticated: true, only_real: true },
     settingsws               : { module: Settings,                   is_authenticated: true },
+    self_exclusionws         : { module: SelfExclusion,              is_authenticated: true, only_real: true },
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: `sel` is before `set`

File: src/javascript/app/base/binary_pages.js [hunk 12]
```diff
@@ -115,36 +125,28 @@ const pages_config = {
     virtualws                : { module: VirtualAccOpening,          not_authenticated: true },
     welcome                  : { module: WelcomePage,                is_authenticated: true, only_virtual: true },
     withdrawws               : { module: PaymentAgentWithdraw,       is_authenticated: true, only_real: true },
+
     'binary-options'         : { module: GetStarted.BinaryOptions },
+    'binary-in-numbers'      : { module: StaticPages.BinaryInNumbers },
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: `binary-i` is after `binary-o`, should be on line 132

File: src/root_files/app/index.html [hunk 13]
```diff
@@ -103,14 +103,14 @@
...
-<!-- End Google Tag Manager -->
+        <!-- Google Tag Manager -->
+        <noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-MZWFF7"
+        height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+        <script data-cfasync="false">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+        '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+        })(window,document,'script','dataLayer','GTM-MZWFF7');</script>
+        <!-- End Google Tag Manager -->
```
Comment (negarn) [inline src/root_files/app/index.html]: please revert this change, we don't need indentations here

File: src/javascript/app/base/binary_loader.js [hunk 14]
```diff
@@ -74,14 +75,15 @@ const BinaryLoader = (() => {
         ContentVisibility.init();
 
         BinarySocket.wait('authorize', 'website_status', 'landing_company').then(() => {
-            const images = $('#content img');
-            if (images) {
-                images.on('load', () => {
+            // first time load.
+            const last_image = $('#content img').last();
```
Comment (negarn) [inline src/javascript/app/base/binary_loader.js]: this still doesn't tell us if the image has loaded, or if it's really the last one, right?
